api/templates/base.html

```diff
@@ -52,6 +52,9 @@
                 <li class="nav-item active">
                     <a class="nav-link" aria-current="page" href="{% url 'scan-report-list' %}">Scan Reports</a>
                 </li>
+                <li class="nav-item active">
+                    <a class="nav-link" aria-current="page" href="{% url 'datasets' %}">Datasets</a>
+                </li>
```

Comment:
Could we perhaps put the Datasets link before the Scan Reports one? Seeing how the Datasets contain all the Scan Reports